pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -140,6 +140,7 @@ class ModelCheckpoint(Callback):
 
     CHECKPOINT_JOIN_CHAR = "-"
     CHECKPOINT_NAME_LAST = "last"
```

Comment:
```suggestion
    JOIN_CHAR = "-"
    FILE_NAME_LAST = "last"
```
lest simplify also this :]

Reply:
similarly, we have `LOGGER_JOIN_CHAR` in loggers too.

Reply:
shall we do both in a separate one? tests are required to be updated too after this.

Reply:
Do I include this in this PR as well? I can, or I can fix this in a separate PR as well.

Reply:
lest Logger do in separate PR